### PR TITLE
New Version of the code with dashboard frontend

### LINKED ISSUE #7: create a frontend design for dashboard

### PR DESCRIPTION
**Pull request by Siose-Patrick**

### CODE REVIEW

**Review by EnterCapslock — APPROVED** [review]
check ko pa

**Review by EnterCapslock — APPROVED** [review]
wala pa naman akong nakitang mali.

File: react-webapp/src/App.css
Comment (Sibayan-ShainahGrace) [inline react-webapp/src/App.css:1]: There's a typo in the font-family name. It should be 'Montserrat' instead of 'Monsterrat.'

**Review by Sibayan-ShainahGrace — CHANGES_REQUESTED** [review]
Mali yung spelling ng font-family name. Dapat "Montserrat," hindi "Monsterrat."

File: react-webapp/src/App.css
Comment (Siose-Patrick) [reply react-webapp/src/App.css:1]: Updated the pull request with the applied the revision

**Review by kwinsiii23 — APPROVED** [review]
okay na, nabago yun wrong spelling. Approved

**Review by EnterCapslock — APPROVED** [review]

**Review by EnterCapslock — APPROVED** [review]
sorry di ko nakita i approve ko nato

**Review by Sibayan-ShainahGrace — APPROVED** [review]
okay na

**Review by EnterCapslock — COMMENTED** [review]
Merge ko na

**Pull request merged**